kubernetes/delete-weblogic-domain-resources.sh

```diff
@@ -53,6 +53,46 @@ cat << EOF
 EOF
 }
 
+#
+# get voyager related resources of one domain
+#
+# Usage:
+#   getVoyagerOfDomain domainName outfilename
+function getVoyagerOfDomain {
+  local voyagerIngressName="ingress.voyager.appscode.com"
+  local domainName=$1
+  local ns=`kubectl get ingress.voyager.appscode.com --all-namespaces | grep $domainName | awk '{ print $1 }'`
+  if [ -n "$ns" ]; then
+    echo $voyagerIngressName $domainName-voyager -n $ns >> $2
+  fi
+}
+
+#
+# get voyager related resources
```

Comment:
Would it be practical to use the 'kubectl label' command to label the voyager generated resources that we don't create directly ourselves?

That would eliminate the need to put voyager-specific code in delete-domain-resources, plus helpfully make the resources show up in global domain label queries... 

      kubectl -n ${namespace} label \
              $RESOURCE_TYPE $RESOURCE_NAME \
              domainUID=$DOMAIN_UID